src/V3AstNodes.h

```diff
@@ -1929,7 +1932,9 @@ class AstSel final : public AstNodeTriop {
         return op1p();
     }  // op1 = Extracting what (nullptr=TBD during parsing)
     AstNode* lsbp() const { return op2p(); }  // op2 = Msb selection expression
+    void lsbp(AstNode* const lsbp) { setOp2p(lsbp); }
     AstNode* widthp() const { return op3p(); }  // op3 = Width
+    void widthp(AstNode* const widthp) { setOp3p(widthp); }
```

Comment:
You probably also need to fix the dtype when setting widthp, probably better (as in: safer in the current implementation) to just create a new Sel and replace.

Reply:
I thought I'd need to save this for intra-assignment delays, but this width should be constant so there's no need. I removed this change.